Header header
geometry_msgs/Point pos
geometry_msgs/Vector3 vel
float target_tm
